DATA/production/qc-postproc-async/zdc.json

```diff
@@ -0,0 +1,226 @@
+              "ZPA2:h_ADC_ZPA_T2",
+              "ZPA3:h_ADC_ZPA_T3",
+              "ZPA4:h_ADC_ZPA_T4",
+              "ZPAS:h_ADC_ZPA_SUM",
+              "ZEM1:h_ADC_ZEM1",
+              "ZEM2:h_ADC_ZEM2",
+              "ZNCC:h_ADC_ZNC_TC",
+              "ZNC1:h_ADC_ZNC_T1",
+              "ZNC2:h_ADC_ZNC_T2",
+              "ZNC3:h_ADC_ZNC_T3",
+              "ZNC4:h_ADC_ZNC_T4",
+              "ZNCS:h_ADC_ZNC_SUM",
+              "ZPCC:h_ADC_ZPC_TC",
+              "ZPC1:h_ADC_ZPC_T1",
+              "ZPC2:h_ADC_ZPC_T2",
+              "ZPC3:h_ADC_ZPC_T3",
+              "ZPC4:h_ADC_ZPC_T4",
+              "ZPCS:h_ADC_ZPC_SUM"
+            ]
+          }
+        ],
+        "dataSourcesTDC": [
+          {
+            "type": "repository",
+            "path": "ZDC/MO/QcZDCRecTask",
+            "names": [
+              "ZNAC:h_TDC_ZNA_TC_V",
+              "ZNAS:h_TDC_ZNA_SUM_V",
+              "ZPAC:h_TDC_ZPA_TC_V",
+              "ZPAS:h_TDC_ZPA_SUM_V",
+              "ZEM1:h_TDC_ZEM1_V",
+              "ZEM2:h_TDC_ZEM2_V",
+              "ZNCC:h_TDC_ZNC_TC_V",
+              "ZNCS:h_TDC_ZNC_SUM_V",
+              "ZPCC:h_TDC_ZPC_TC_V",
+              "ZPCS:h_TDC_ZPC_SUM_V"
+            ]
+          }
+        ],
+        "initTrigger": [
+          "userorcontrol"
+        ],
+        "updateTrigger": [
+          "newobject:qcdb:ZDC/MO/Rec/h_ADC_ZNA_TC"
+        ],
+        "stopTrigger": [
+          "userorcontrol"
+        ]
+      },
+    "ZDCQuality": {
+        "active": "true",
+        "className": "o2::quality_control_modules::common::QualityTask",
+        "moduleName": "QualityControl",
+        "detectorName": "ZDC",
+        "qualityGroups": [
+          {
+            "name": "global",
+            "title": "GLOBAL ZDC QUALITY",
+            "path": "ZDC/QO",
+            "ignoreQualitiesDetails": [
+              "Null",
+              "Good",
+              "Medium",
+              "Bad"
+            ],
+            "inputObjects": [
+              {
+                "name": "ZDCQuality/ZDCQuality",
+                "title": "ZDC Quality",
+                "messageBad": "Inform on-call immediately",
+                "messageMedium": "Add bookkeeping entry",
+                "messageGood": "All checks are OK",
+                "messageNull": "Some histograms are empty!!!"
+              }
+            ]
+          },
+          {
+            "name": "details",
+            "title": "ZDC DETAILS",
+            "path": "ZDC/QO",
+            "ignoreQualitiesDetails": [],
+            "inputObjects": [
+              {
+                "name": "QcZDCRecCheck",
+                "title": "Rec check"
+              }
+            ]
+          }
+        ],
+        "initTrigger": [
+          "userorcontrol"
+        ],
+        "updateTrigger": [
+          "newobject:qcdb:ZDC/QO/ZDCQuality/ZDCQuality"
+        ],
+        "stopTrigger": [
+          "userorcontrol"
+        ]
+      }
+    },
+      "checks": {
```

Comment:
The indentation seems to be not correct.

Reply:
done